app/src/main/java/com/example/myapplication/ui/home/model/PostedItems.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.myapplication.ui.home.model
+
+data class PostedItem(
+    val profile: UserProfile,
+    val imageUri: String,
+    val commentsList: List<Comment>,
+    val likesCount: Int,
+    val isLiked: Boolean,
+    val timeStamp: String,
+    val imageDescription: String
```

Comment:
Do czego będzie służyć to pole?

Reply:
Czy muszę zmienić nazwy czy uważasz ich za niepotrzebne ?

Reply:
Pytam, ponieważ ImageView posiadają atrybut description, wykorzytywany dla osób niepełnosprawnych, jak również na trzeciej zakładce pojawia się description posta. Także nie wiem do czego zamierzałeś użyć to pole `val imageDescription: String` :)
![image](https://user-images.githubusercontent.com/1727906/114174594-1d60c980-9939-11eb-9595-84a936e55901.png)

Reply:
No właśnie i chciałem użyć jak opis postu, z przykładu "goodbye chaose".

Reply:
Tak rozumiem, że lepiej zmienić na "postDesription"?

Reply:
Tak, myślę że lepiej zmienić nazwę pola żeby rozwiać wątpliwości. Wystarczy `title` lub `description`